src/content/docs/apis/nerdgraph/examples/nerdgraph-api-secret-management-service.mdx

```diff
@@ -0,0 +1,255 @@
+    
+    ```json
+    {
+    "data": {
+        "secretsManagementCreateSecret": {
+        "createdAt": "2021-09-29T10:00:00Z",
+        "latestVersion": "1",
+        "updatedAt": "2021-09-29T10:00:00Z"
+        }
+    }
+    }
+    ```
+
+
+### Purge secrets[#purge-secrets]
+
+The `secretsManagementPurgeSecret` mutation allows you to delete secrets from the Secrets Management Service. You can delete a specific version of a secret or all versions of a secret.
+
+#### Input Parameters
+
+* `accountId` (String, required): The account ID associated with the secret.
+* `key` (String, required): The key for the secret.
+* `namespace` (String, optional): The namespace to which the secret belongs.
+* `purgeAll` (Boolean, required): If true, all versions of the secret are deleted. 
+* `version` (String, optional): The version of the secret to delete.
+
+<Callout variant="tip">
+    * If `purgeAll` is true, the `version` parameter is null and ignored.
+    * valid combination of `purgeAll` and `version` : `true` & `null`, `false` & `non-null`.
+</Callout>
+
+#### Response
+
+* `key` (String): The key of the created secret.
+* `latestVersion` (String): The latest version available corresponding to the key. If `purgeAll` is true or no version is available after purge, latestVersion will be null.
+* `tags` (Array): Metadata associated with the secret.
+
+#### Sample query:
+
+```graphql
+
+mutation {
+  secretsManagementPurgeSecret(
+    accountId: 12345678
+    key: "test_user_key"
+    namespace: ""
+    purgeAll: true
+    version: ""
+  ) {
+    key
+    latestVersion
+    tags
+  }
+}
+
+```
+
+#### Sample response:
+
+```json
+{
+  "data": {
+    "secretsManagementPurgeSecret": {
+      "latestVersion": null
+    }
+  },
+}
+```
+
+
+### Retrive a secret[#retrieve-secret]
+
+The `secretsManagementRetrieveSecret` mutation allows you to access the value of stored secrets.
+
+#### Input Parameters
+
+* `accountId` (String, required): The account ID associated with the secret.
+* `namespace` (String, optional): The namespace to which the secret belongs.
+* `key` (String, required): The key of the secret that needs to be retrieved.
+* `version`(String, required): The version of the secret that needs to be retrieved.
+
+#### Response
+
+* `value` (SecureValue, required): The value of the secret.
+* `tags` (Array): Metadata associated with the secret.
+* `createdAt` (String): The time the secret was created.
+
+#### Sample query:
+
+```graphql
+query {
+  secretsManagementRetrieveSecret(
+    namespace: "test"
+    key: "test_user_key"
+    version: ""
+  ) {
+    value
+    tags
+    createdAt
+  }
+}
+```
+
+#### Sample response:
+
+```json
+{
+  "data": {
+    "secretsManagementRetrieveSecret": {
+      "value": "1234567890",
+      "tags": [],
+      "createdAt": "2021-09-29T10:00:00Z"
+    }
+  }
+}
+```
+
+
+### Update a secret[#update-secret]
+
+The `secretsManagementUpdateSecret` mutation allows you to update the value of existing secrets.
```

Comment:
Check what guidance we have in our style guide for allows vs. enables.

Preferably we should use enables or lets you. Allows indicates permission. I am not sure we mean that. I have seen this word used a few times in the topic. Based on what you find out, please change across instances.